src/DependencyInjection/Configuration.php

```diff
@@ -28,6 +28,10 @@ private function addConverterSection(ArrayNodeDefinition $rootNode): void
     {
         $rootNode
             ->children()
+                ->scalarNode('converterSuffix')
+                    ->info('consolidated suffix for all converters')
+                    ->defaultValue('')
```

Comment:
This is a breaking change. The default value should be the current one.

```suggestion
                    ->defaultValue('Converter')
```